tests/unit/testsuite/Bolt/Boltpay/Helper/GeneralTraitTest.php

```diff
@@ -0,0 +1,453 @@
+<?php
+require_once('TestHelper.php');
+require_once('CouponHelper.php');
```

Comment:
Please remove require_once

Reply:
These are currently required.  Your request will be covered here:
https://app.asana.com/0/941895179897714/1153588938387929